smart-cards/src/main/java/gov/va/api/health/smartcards/Dstu2OpenApi.java

```diff
@@ -0,0 +1,20 @@
+package gov.va.api.health.smartcards;
+
+import gov.va.api.health.smartcards.patient.SmartCardsPatientApi;
+import io.swagger.v3.oas.annotations.ExternalDocumentation;
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.annotations.info.Info;
+import io.swagger.v3.oas.annotations.servers.Server;
+import javax.ws.rs.Path;
+
+@OpenAPIDefinition(
+    info = @Info(title = "SMART Health Cards", version = "v1"),
+    servers = {
+      @Server(url = "https://sandbox-api.va.gov/services/fhir/v0/dstu2/", description = "Sandbox")
+    },
+    externalDocs =
+        @ExternalDocumentation(
+            description = "Smart Cards Implementation Guide",
+            url = "https://smarthealth.cards"))
+@Path("/")
+public interface Dstu2OpenApi extends SmartCardsPatientApi {}
```

Comment:
Let's rename `SmartCardsPatientApi` to `R4SmartCardsPatientApi` since it's using r4 `Parameters`

`Dstu2OpenApi extends R4SmartCardsPatientApi` looks odd, but that is how it's set up...